stl/inc/ranges

```diff
@@ -1433,6 +1463,93 @@ namespace ranges {
         inline constexpr _Istream_fn<_Ty> istream;
     } // namespace views
 
+#if _HAS_CXX23
+    namespace _Pipe {
+        template <class _Ty>
+        void _Derived_from_range_adaptor_closure_impl(const volatile range_adaptor_closure<_Ty>&);
+
+        template <class _Ty>
+        concept _Derived_from_range_adaptor_closure = requires(const volatile _Ty& __obj) {
+            _Derived_from_range_adaptor_closure_impl(__obj);
+        };
+
+        template <class _Ty>
+        concept _Is_range_adaptor_common = derived_from<remove_cvref_t<_Ty>,
+            range_adaptor_closure<remove_cvref_t<_Ty>>> && _Derived_from_range_adaptor_closure<_Ty> && !range<_Ty>;

```

Comment:
This is currently had to read. Could you try to add a `//` before the `&&` so that you can force the first derived_from to a single line?